src/main/java/de/dennisguse/opentracks/io/file/importer/KmzTrackImporter.java

```diff
@@ -71,7 +66,7 @@ public KmzTrackImporter(Context context, Uri uriFile) {
     }
 
     @Override
-    public Track.Id importFile(InputStream inputStream) throws SAXException, SQLiteConstraintException, IOException, ParserConfigurationException {
+    public Track.Id importFile(InputStream inputStream) throws ImportParserException, ImportAlreadyExistsException {
```

Comment:
Throws declaration of the two new Exceptions is not necessary, because they are unchecked Exceptions.

Reply:
Ops!!! Yes, of course.